relay-general/src/store/transactions/processor.rs

```diff
@@ -2568,4 +2568,11 @@ mod tests {
         "db.sql.query",
         ""
     );
+
+    span_description_test!(
+        span_description_dont_scrub_nulls,
+        "SELECT * FROM table WHERE deleted_at IS NULL",
```

Comment:
Question about placeholders in general: Since `*` is a [special character](https://www.postgresql.org/docs/current/sql-syntax-lexical.html) in SQL, should we use a different placeholder when we scrub something?

Reply:
Good comment! I addressed this in a follow-up: https://github.com/getsentry/relay/pull/2126.